presto-docs/src/main/sphinx/release/release-0.290.rst

```diff
@@ -0,0 +1,125 @@
+=============
+Release 0.290
+=============
+
+**Highlights**
+==============
+
```

Comment:
The only thing left is for someone to write any needed entries in **Highlights**.